packages/itwin/property-grid/.eslintrc.js

```diff
@@ -1,26 +1,18 @@
 /*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
-*--------------------------------------------------------------------------------------------*/
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
 
 module.exports = {
   plugins: ["@itwin", "unused-imports"],
-  extends: ["plugin:@itwin/ui", "plugin:react/jsx-runtime"],
+  extends: ["plugin:@itwin/ui", "plugin:react/jsx-runtime", "prettier"],
   rules: {
     "@itwin/no-internal": ["error"],
-    "@typescript-eslint/no-floating-promises": [
-      "error",
-      { "ignoreIIFE": true }
-    ],
     "@typescript-eslint/consistent-type-imports": "error",
     "no-duplicate-imports": "off",
     "import/no-duplicates": "error",
     "object-curly-spacing": ["error", "always"],
-    "@typescript-eslint/no-unused-vars": "off",
-    "unused-imports/no-unused-imports": "error",
-    "unused-imports/no-unused-vars": [
-      "error",
-      { "vars": "all", "varsIgnorePattern": "^_", "args": "after-used", "argsIgnorePattern": "^_" }
-    ]
-  }
-}
+
```

Comment:
why remove the `no-unused-*` rules?

Reply:
Forgot to revert after testing. Added back.